DefaultRollingUpgradeTestIT.testDefaultSetting failures

### Describe the bug

`DefaultRollingUpgradeTestIT.testDefaultSetting()` is failing occasionally on CI.

```
java.lang.RuntimeException: java.lang.RuntimeException: java.lang.IllegalStateException: Servers did not cluster within 30 seconds, assuming error
	at org.infinispan.server.test.core.rollingupgrade.RollingUpgradeConfigurationBuilder.lambda$new$0(RollingUpgradeConfigurationBuilder.java:39)
	at org.infinispan.server.test.core.rollingupgrade.RollingUpgradeHandler.exceptionEncountered(RollingUpgradeHandler.java:339)
	at org.infinispan.server.test.core.rollingupgrade.RollingUpgradeHandler.performUpgrade(RollingUpgradeHandler.java:347)
	at org.infinispan.server.rollingupgrade.DefaultRollingUpgradeTestIT.testDefaultSetting(DefaultRollingUpgradeTestIT.java:39)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
Caused by: java.lang.RuntimeException: java.lang.IllegalStateException: Servers did not cluster within 30 seconds, assuming error
	at org.infinispan.server.test.core.rollingupgrade.RollingUpgradeConfigurationBuilder.lambda$new$0(RollingUpgradeConfigurationBuilder.java:39)
	at org.infinispan.server.test.core.rollingupgrade.RollingUpgradeHandler.exceptionEncountered(RollingUpgradeHandler.java:339)
	at org.infinispan.server.test.core.rollingupgrade.RollingUpgradeHandler.completeUpgrade(RollingUpgradeHandler.java:329)
	at org.infinispan.server.test.core.rollingupgrade.RollingUpgradeHandler.performUpgrade(RollingUpgradeHandler.java:345)
	... 4 more
Caused by: java.lang.IllegalStateException: Servers did not cluster within 30 seconds, assuming error
	at org.infinispan.server.test.core.rollingupgrade.RollingUpgradeHandler.upgradeNewNode(RollingUpgradeHandler.java:318)
	at org.infinispan.server.test.core.rollingupgrade.RollingUpgradeHandler.completeUpgrade(RollingUpgradeHandler.java:325)
	... 5 more
```

https://github.com/infinispan/infinispan/pull/15891/checks?check_run_id=52883853220

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### Infinispan version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/server/rollingupgradetests/src/test/java/org/infinispan/server/rollingupgrade/RollingUpgradeTestUtil.java
+++ b/server/rollingupgradetests/src/test/java/org/infinispan/server/rollingupgrade/RollingUpgradeTestUtil.java
@@ -9,7 +9,7 @@ import java.util.Objects;
 
 public class RollingUpgradeTestUtil {
    public static String getFromVersion() {
-      return System.getProperty(INFINISPAN_ROLLING_UPGRADE_FROM_VERSION, "16.0.0.Dev04");
+      return System.getProperty(INFINISPAN_ROLLING_UPGRADE_FROM_VERSION, "16.0.0.Dev06");
    }
 
    public static String getToVersion() {

